db/migrate/20210930100602_add_search_terms_to_users.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class AddSearchTermsToUsers < ActiveRecord::Migration[6.0]
+  def up
+    execute <<-SQL.squish
+      ALTER TABLE users
+      ADD COLUMN search_terms tsvector GENERATED ALWAYS AS (
+        to_tsvector('french', coalesce(first_name, '')) ||
+        to_tsvector('french', coalesce(birth_name,'')) ||
+        to_tsvector('french', coalesce(last_name, '')) ||
+        to_tsvector('french', coalesce(email, '')) ||
+        to_tsvector('french', coalesce(phone_number_formatted, '')) ||
+        to_tsvector('french', coalesce(phone_number, ''))
+      ) STORED;
```

Comment:
J’ai l’impression qu’on peut écrire `add_column :users, :search_terms, :tsvector` plutôt que de faire du sql. (J’ai vraiment pas envie de faire du sql😅)

C’est pour réunir tous les search_terms en une seule colonne? [Ici](https://thoughtbot.com/blog/optimizing-full-text-search-with-postgres-tsvector-columns-and-triggers), il font une colonne “normale” plutôt que `GENERATED`, mais une fonction trigger. Est-ce que ça serait plus simple?